pythonFiles/unittestadapter/discovery.py

```diff
@@ -122,8 +120,15 @@ def post_response(
     start_dir, pattern, top_level_dir = parse_unittest_args(argv[index + 1 :])
 
     # Perform test discovery.
-    testPort = int(os.environ.get("TEST_PORT", DEFAULT_PORT))
+    testPort = int(os.environ.get("TEST_PORT"))
```

Comment:
```suggestion
    testPort = int(os.environ["TEST_PORT"]) if "TEST_PORT" in os.environ else None
```

I think `int(None)` would turn into `0` and not pass the `testPort is None` condition below.

Reply:
Never mind, `int(None)` errors, so this probably needs the same treatment as `run_test_ids_port_int` below.